xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/NfTokenObject.java

```diff
@@ -0,0 +1,43 @@
+package org.xrpl.xrpl4j.model.client.accounts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+import org.xrpl.xrpl4j.model.transactions.NfTokenId;
+
+import java.util.Optional;
+
+/**
+ * Structure of an NFToken stored on the ledger.
+ */
+@Value.Immutable
+@JsonSerialize(as = ImmutableNfTokenObject.class)
+@JsonDeserialize(as = ImmutableNfTokenObject.class)
+public interface NfTokenObject {
+
+  /**
+   * Construct a builder for this class.
+   *
+   * @return An {@link ImmutableNfTokenObject.Builder}.
+   */
+  static ImmutableNfTokenObject.Builder builder() {
+    return ImmutableNfTokenObject.builder();
+  }
+
+  /**
+   * The unique TokenID of the token.
+   *
+   * @return The unique TokenID of the token.
+   */
+  @JsonProperty("TokenID")
```

Comment:
Shouldn't this property be named `NFTokenID` as in other places? I've tried listing account NFTs and it threw an exception because `TokenID` was not part of JSON response.

Reply:
Thank you for pointing that out!